_sass/site/blog/post/_post-code.scss

```diff
@@ -0,0 +1,3 @@
+ul, ol {
```

Comment:
po co ten styl? taki kod znajduje się w post-lists

Reply:
to jest in-progress, skopiowałem styl ale nic jeszcze w nim nie robiłem